src/signature.rs

```diff
@@ -194,27 +198,11 @@ impl<'a> Signature {
 }
 
 /// A signature verification algorithm.
```

Comment:
I think we need to add a note that the trait isn't intended to be implemented by things outside the *ring* crate.